src/browser/createLogger.ts

```diff
@@ -4,8 +4,20 @@ import { gradient } from './gradient';
 import type { Options, } from '../types';
 import { LOG_TYPES } from './constants'
 
+function validateOptions(options: Options): Options {
+   const validatedOptions = { ...options };
+   if (options.labels && typeof options.labels !== 'object') {
+      throw new Error('Labels must be an object');
+   }
+   if (options.level && typeof options.level !== 'string') {
+      throw new Error('Level must be a string');
+   }
+   return validatedOptions;
+}
+
 export function createLogger(options: Options = {},) {
-   return rawCreateLogger(options, {
+   const validatedOptions = validateOptions(options);
```

Comment:
how about move the validate to rawCreateLogger ?

Reply:
done